src/Command/PurgeCommand.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+namespace Terminal42\LeadsBundle\Command;
+
+use Contao\CoreBundle\Framework\ContaoFrameworkInterface;
+use Contao\CoreBundle\Monolog\ContaoContext;
+use Contao\File;
+use Contao\FilesModel;
+use Contao\System;
+use Contao\Validator;
+use Doctrine\DBAL\Connection;
+use Psr\Log\LogLevel;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Filesystem\Filesystem;
+use Terminal42\LeadsBundle\Leads;
+use \Exception;
+
+class PurgeCommand extends Command
+{
+    /**
+     * @var ContaoFrameworkInterface
+     */
+    private $framework;
+
+    /**
+     * @var Connection
+     */
+    private $db;
+
+    /**
+     * @var Filesystem
+     */
+    private $fs;
+
+    /**
+     * Constructor.
+     *
+     * @param ContaoFrameworkInterface $framework
+     * @param Connection $db
+     * @param Filesystem $fs
+     */
+    public function __construct(ContaoFrameworkInterface $framework, Connection $db, Filesystem $fs = null)
+    {
+        if (null === $fs) {
+            $fs = new Filesystem();
+        }
+
+        $this->framework = $framework;
+        $this->db = $db;
+        $this->fs = $fs;
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('leads:purge')
```

Comment:
Should prefix that with Contao? `contao:leads:purge` or `contao:purge-leads`?
I don't think we've *decided* on a generic command name rule. /cc @terminal42/pilots

Reply:
just focused on the `leads:export` command - how should the naming be?